src/openforms/config/migrations/0001_initial_to_v250.py

```diff
@@ -24,6 +26,17 @@
 import openforms.utils.translations
 
 
+# Function removed from the code, moved here to not break the migration:
+def validate_payment_order_id_prefix(value: str):
+    value = value.replace("{year}", "")
+    if value and not value.isalnum():
+        raise ValidationError(
+            _(
+                "Prefix must be alpha numeric, no spaces or special characters except {year}"
+            )
+        )
```

Comment:
I think you can remove the implementation entirely in historical migrations with just a `pass`, they don't do anything anyway.